app.js

```diff
@@ -0,0 +1,353 @@
+/*global alert ,confirm ,window ,setInterval ,setTimeout ,document ,clearInterval*/
+    //  Get user info
+(function getUserInfo() {
+    'use strict';
+    var url = window.location.href,
+        userName,
+        userAge,
+        greetDate,
+        userNamePlace,
+        getTimeNow;
+
+    //  if game page has no data(string) in url, keep moving.
+    //  Otherwise, declare user's name and age variables.
+    if ("string" === typeof url.split("?")[1]) {
+
+        userName = url.split("?")[1].split("=")[1].split("&")[0];
```

Comment:
If you see a similar piece of code written multiple times, then something can be improved:

```js
var urlParts = url.split("?")[1],
      // from 'http://blabla.com/script/?userName=shady&userAge=21
      userName = urlParts.match( /userName=([^&]+)/ )[1],
      userAge = urlParts.match( /userAge=([^&]+)/ )[1];
```